comm.cpp

```diff
@@ -320,17 +321,24 @@ int UdpEndpoint::open(const char *ip, unsigned long port)
     sockaddr.sin_addr.s_addr = inet_addr(ip);
     sockaddr.sin_port = htons(port);
 
-    if (setsockopt(fd, SOL_SOCKET, SO_BROADCAST, &broadcast_val, sizeof(broadcast_val))) {
-        log_error_errno(errno, "Error enabling broadcast in socket (%m)");
-        goto fail;
+    if (to_bind) {
```

Comment:
but you want to *connect* to that IP+port, not bind... the simulator should be running on a known port, not mavlink-router.